element_interface/prairieviewreader.py

```diff
@@ -110,23 +89,57 @@ def get_pv_metadata(pvtiffile: str) -> dict:
 
     um_height = um_width = float(px_height) * um_per_pixel
 
-    z_min = float(
-        root.findall(
-            ".//Sequence/[@cycle='1']/Frame/PVStateShard/PVStateValue/[@key='positionCurrent']/SubindexedValues/SubindexedValue/[@subindex='0']"
-        )[0].attrib.get("value")
-    )
-    z_max = float(
-        root.findall(
-            ".//Sequence/[@cycle='1']/Frame/PVStateShard/PVStateValue/[@key='positionCurrent']/SubindexedValues/SubindexedValue/[@subindex='0']"
-        )[-1].attrib.get("value")
+    # x and y coordinate values for the center of the field
+    x_field = float(
+        root.find(
+            ".//PVStateValue/[@key='currentScanCenter']/IndexedValue/[@index='XAxis']"
+        ).attrib.get("value")
     )
-    z_step = float(
+    y_field = float(
         root.find(
-            ".//PVStateShard/PVStateValue/[@key='micronsPerPixel']/IndexedValue/[@index='ZAxis']"
+            ".//PVStateValue/[@key='currentScanCenter']/IndexedValue/[@index='YAxis']"
         ).attrib.get("value")
     )
-    z_fields = np.arange(z_min, z_max + 1, z_step)
-    assert z_fields.size == n_depths
+
+    if root.find(".//Sequence/[@cycle='1']/Frame/PVStateShard/PVStateValue/[@key='positionCurrent']/SubindexedValues/[@index='ZAxis']") is None:
+
+        z_fields = np.float64(
+            root.find(
+                ".//PVStateValue/[@key='positionCurrent']/SubindexedValues/[@index='ZAxis']/SubindexedValue").attrib.get("value")
+        )
+        n_depths = 1
+        assert z_fields.size == n_depths
+        bidirection_z = False
+        
```

Comment:
My flake8 linter caught some whitespace here (W293) we could easily clean up. Please also run black. My version 22.3.0 made some changes.
```console
 flake8 --max-line-length=88 --max-complexity=18 --select=B,C,E,F,W,T4,B9 --ignore
=E203,E501,W503,W605
```

Reply:
Thanks, Chris. I'm not seeing any more whitespace issues from pylint after running Black.